features/device-config/monitor/src/main/java/org/opennms/features/deviceconfig/monitors/DeviceConfigMonitor.java

```diff
@@ -140,10 +142,17 @@ public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
 
         final Date lastRun = new Date(getKeyedLong(parameters, LAST_RETRIEVAL, 0L));
         final String cronSchedule = getKeyedString(parameters, DeviceConfigConstants.SCHEDULE, DeviceConfigConstants.DEFAULT_CRON_SCHEDULE);
```

Comment:
Should `never` be the default?

Reply:
Good question, but nothing that I want to decide in this PR.